qiskit/pulse/pulse_lib/discrete.py

```diff
@@ -166,18 +239,32 @@ def sin(duration: int, amp: complex, freq: float = None,
 
 def gaussian(duration: int, amp: complex, sigma: float, name: Optional[str] = None,
              zero_ends: bool = True) -> 'SamplePulse':
-    r"""Generates unnormalized gaussian `SamplePulse`.
+    r"""Generates unnormalized gaussian :class:`~qiskit.pulse.SamplePulse`.
+
+    For :math:`A=` ``amp`` and :math:`\sigma=` ``sigma``, applies the `midpoint` sampling strategy
+    to generate a discrete pulse sampled from the continuous function:
 
-    Centered at `duration/2` and zeroed at `t=0` and `t=duration` to prevent large
-    initial/final discontinuities.
+    .. math::
 
-    Applies `midpoint` sampling strategy to generate discrete pulse from continuous function.
+        f(x) = A\exp\left(\left(\frac{x - \mu}{\sigma}\right)^2 \right),
```

Comment:
I think this should be 
```suggestion
        f(x) = A\exp\left(\left(\frac{x - \mu}{2\sigma}\right)^2 \right),
```